src/app/components/mno-local-products/mno-local-product.html

```diff
@@ -70,7 +70,8 @@ <h2>{{ 'mno_enterprise.templates.dashboard.marketplace.show.price' | translate }
                 <b>{{ 'mno_enterprise.templates.dashboard.marketplace.show.free_trial_duration' | translate }}</b>: {{vm.product.free_trial_duration}} {{vm.product.free_trial_unit}}
               </li>
               <li ng-repeat="plan in vm.product.pricing_plans">
-                <b>{{plan.name}}</b>: {{plan.description}} <br>
+                <b>{{plan.name}}</b>: {{plan.description}}   <i ng-show="plan.quote_based" class="fa fa-question-circle" uib-tooltip="{{ 'mno_enterprise.templates.dashboard.marketplace.show.quote_based' | translate }}"></i>
```

Comment:
Same thing with the extra spaces before the opening `<i>` here unless they are intentional?